packages/frontend-shared/src/utils/isRunMode.ts

```diff
@@ -1 +1 @@
-export const isRunMode = window.__CYPRESS_MODE__ === 'run' && window.top === window
+export const isRunMode = window.location.href.includes('CY_IN_CY_SIMULATE_RUN_MODE') || (window.__CYPRESS_MODE__ === 'run' && window.top === window)
```

Comment:
Can we extract `CY_IN_CY_SIMULATE_RUN_MODE` as a constant? Feels like a we're adding in more magic strings

Reply:
Constant is a good idea, will do.